src/ensembl/src/shared/components/tabs/Tabs.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import styles from './Tabs.scss';
+
+export type Tab = {
+  title: string;
+  isDisabled?: boolean;
+};
+
+export type TabsProps = {
+  tabs: Tab[];
+  classNames?: {
+    default?: string;
+    disabled?: string;
+    selected?: string;
+    tabsContainer?: string;
+  };
+  selectedTab: string;
+  selectTab: (selectedTab: string) => void;
+};
+
+export const Tabs = (props: TabsProps) => {
+  const [selectedTab, setSelectedTab] = useState(props.selectedTab);
+
+  const getTabClassNames = (tab: Tab) => {
+    const defaultClassNames = classNames(props.classNames?.default, styles.tab);
```

Comment:
here and below — same as in comment above about the order of the arguments

Reply:
That's done

Reply:
Github still shows me the reverse order of the arguments:
`classNames(props.classNames?.default, styles.tab);`

Reply:
I missed just this one place. It's done now